src/rebuild/scan.c

```diff
@@ -776,12 +783,14 @@ rebuild_scanner(void *data)
 		D_GOTO(out, rc);
 	}
 
-	rpt_get(rpt);
-	rc = dss_ult_create(rebuild_objects_send_ult, rpt, DSS_XS_SELF,
-			    0, 0, &ult_send);
-	if (rc != 0) {
-		rpt_put(rpt);
-		D_GOTO(out, rc);
+	if (rpt->rt_rebuild_op != RB_OP_RECLAIM) {
```

Comment:
Why it is different from other cases, such as reint. As my understand, they all need to scan and send the objects to be rebuilt.

Reply:
reclaim does not send objects to other target, since it will scan the whole obj table to find out if the shard still belong to the current object with the current map, otherwise reclaim it. So no need send objects to other ULT. 

see rebuild_obj_scan_cb().